### PR TITLE
Add missing border-radius in collection table

### LINKED ISSUE #33744: Missing border-radius in collection table
### Describe the bug

image

### To Reproduce

1. Create some models/questions/dashboards and add them to "Our analytics"
2. Open "Our analytics" and look at the table corners

### Expected behavior

There is no white area around table corners

### Information about your Metabase installation

v47, ed6c220

### Severity

P3

### PR DESCRIPTION
**Pull request by ibedwi**
Closes https://github.com/metabase/metabase/issues/33744

#### Root cause

In the `Table` style, the `radius` properties are applied to the `th` and `tr`, while the `background-color` property is applied to the `table` element. 

#### Approach

In this PR the `border-radius` property also added to the `table` element.

### How to verify

1. Create some models/questions/dashboards and add them to "Our analytics"
2. Open "Our analytics" and look at the table corners

### Demo

_Upload a demo video or before/after screenshots if sensible or remove the section_

(to make the difference clearer, the app's background is changed to `blue`)

#### Before

CleanShot 2023-11-05 at 19 57 26@2x

#### After

CleanShot 2023-11-05 at 19 57 57@2x

### Checklist

- [ ] Tests have been added/updated to cover changes in this PR

### CODE REVIEW

**Comment by ranquild** [comment]
FE changes LGTM, waiting for @kdoh approval

**Review by ranquild — APPROVED** [review]

**Comment by nemanjaglumac** [comment]
Thank you once again @ibedwi

**Comment by nemanjaglumac** [comment]
Added no-backport since I couldn't find this problem in the latest `47.7` JAR.

**Pull request merged**
